### PR TITLE
Syntax-highlight ALL the things.

### PR DESCRIPTION
**Pull request by hjwp**
following #335, this puts all the other codeblocks into syntax-highlighted chunks.

### CODE REVIEW

**Comment by hjwp** [comment]
hey, am i being antisocial by only applying these changes to the `en` files, and not the other languages?

File: en/css/README.md [hunk 1]
```diff
@@ -45,9 +48,13 @@ First, we need to create a directory to store our static files in. Go ahead and
 
 Open up the `mysite/settings.py` file, scroll to the bottom of it and add the following lines:
 
-    STATICFILES_DIRS = (
-        os.path.join(BASE_DIR, "static"),
-    )
+<!--TODO: this will conflict with the STATIC_ROOT specified earlier -->
```
Comment (bmispelon) [inline en/css/README.md]: Was this meant to be part of a different pull request?

File: en/django_start_project/README.md [hunk 2]
```diff
@@ -50,16 +50,13 @@ Let's ignore the other files for now - we won't change them. The only thing to r
 
 Let's make some changes in `mysite/settings.py`. Open the file using the code editor you installed earlier.
 
-It would be nice to have the correct time on our website. Go to the [wikipedia timezones list](http://en.wikipedia.org/wiki/List_of_tz_database_time_zones) and copy your relevant time zone (TZ). (eg. `Europe/Berlin` )
+It would be nice to have the correct time on our website. Go to http://en.wikipedia.org/wiki/List_of_tz_database_time_zones and copy your relevant time zone (TZ). (eg. `Europe/Berlin` )
```
Comment (bmispelon) [inline en/django_start_project/README.md]: These changes are not just syntax-highlight related and probably belong in another PR.

File: en/css/README.md (hunk 1 above)
Comment (hjwp) [inline en/css/README.md]: oops, sorry.  will review the PR line-by-line and make sure it's just syntax-highlights.

**Comment by hjwp** [comment]
have removed the superfluous bits in #345
